cypress/component/Page.spec.tsx

```diff
@@ -1,13 +1,66 @@
 import React from 'react'
 import type { PageSidebarProps } from '@toptal/picasso'
 import { Container, Menu, Page, Typography } from '@toptal/picasso'
+import type { BreakpointKeys } from '@toptal/picasso-provider/Picasso/config'
+import { PicassoBreakpoints } from '@toptal/picasso-provider/Picasso/config'
 
 const component = 'Page'
 const containerHeight = '30rem'
 
+/**
+ * Produces an array of checkpoints – screen size values, needed for covering
+ * all existing breakpoints in tests (checkpoints are different from breakpoints
+ * by certain offset).
+ *
+ * @returns {number[]} Array of checkpoints
+ */
+const getCheckpoints = () => {
+  const offset = 5
```

Comment:
I would be even safer and use `1` as the offset, as having it with `5` would miss off-by-one errors. If possible I would even go as far as having a decimal offset, something like 0.01, or similar
```suggestion
  const offset = 1
```

Reply:
@augustobmoura Changed to `1`, thanks!